packages/@aws-cdk/aws-ec2-alpha/README.md

```diff
@@ -231,6 +231,63 @@ new Route(this, 'DynamoDBRoute', {
 });
 ```
 
+## VPC Peering Connection
+
+VPC peering connection allows you to connect two VPCs and route traffic between them using private IP addresses. The VpcV2 construct supports creating VPC peering connections through the `VPCPeeringConnection` construct from the `route` module.
+
+This is an example of how to create a VPC peering connection between two VPCs. The process involves 3 main steps:
+
+**Step 1: Acceptor Account Stack**
+
+Create a restrictive IAM role in the acceptor VPC account. This role will be used to grant limited permissions for accepting the peering request from the requestor account.
+
+```ts
+// Step 1: Acceptor Account Stack
+const stack = new Stack();
+
+const acceptorVpc = new VpcV2(this, 'VpcA', {
+  primaryAddressBlock: IpAddresses.ipv4('10.0.0.0/16'),
+});
+
+const acceptorRoleArn = acceptorVpc.createAcceptorVpcRole('987654321098')
```

Comment:
do not use real account Ids

Reply:
this one is fake, starts from 9 and goes to 0 😄

Reply:
:) .. It was looking like a real one, I did not carefully read it to be honest .. my bad :)